src/Common/Internal/Http/HttpClient.php

```diff
@@ -84,6 +84,8 @@ public function __construct(
             Resources::USE_BRACKETS => true,
             Resources::SSL_VERIFY_PEER => false,
             Resources::SSL_VERIFY_HOST => false,
+            'proxy_host' => "localhost",
```

Comment:
I've tried this one on Windows

```bat
setlocal
set HTTP_PROXY="http://localhost:8888"
php my.php
endlocal
```

It doesn't work.

The changes shouldn't break the SDK since they are commented.